src/NServiceBus.Transport.RabbitMQ.Tests/ApprovalTestConfig.cs

```diff
@@ -1,8 +1,6 @@
-﻿#if NET452
-using ApprovalTests.Reporters;
-#if DEBUG
+﻿using ApprovalTests.Reporters;
+#if NET452
 [assembly: UseReporter(typeof(DiffReporter), typeof(AllFailingTestsClipboardReporter))]
```

Comment:
This config is currently only used under Debug for NET452. With this change we will now use it for both Debug and Release. Does that matter? (I've no idea what it does.)

Reply:
As discussed, this is fine, but after further review we had to swap the .NET Core lib to `QuietReporter`.